src/userProfile/profile.style.js

```diff
@@ -30,8 +31,17 @@ export const ProfileForm = styled.form`
   height: 500px;
 `;
 
+export const LockedInputStyle = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 2px solid black;
+`;
+
 export const AvatarsStyles = styled.div`
   width: 700px;
+  height: 67vh;
 `;
```

Comment:
Using `vh` for something that is not the body, or with a value different than 100 can bring a lot of problems. Can we use a unit that doesn't depend on the view port height?

Reply:
sure